client/components/oauth/oauth.tsx

```diff
@@ -23,20 +25,34 @@ interface Props {
 }
 
 const OAuth = ({ history }: Props) => {
-  const [, setCookie] = useCookies(['jwt']);
+  const [cookie, setCookie] = useCookies(['jwt']);
 
   const dispatch = useDispatch();
 
   const oAuthSuccess = (response: GoogleLoginResponse) => {
     setCookie('jwt', response.getAuthResponse().id_token, { path: '/' });
-    dispatch(setUser(response));
+    const decodedJwt = jwtDecode(cookie.jwt);
```

Comment:
I think it would make more sense to assign `id_token` in local scope and use that instead of `cookie`. I don't *think* the reference to `cookie` will be updated immediately after `setCookie` so I'm not clear how this works.